### DISCUSSION TITLE
'Esc key' does not get me out of address bar

### DISCUSSION THREAD

**Discussion by nanoraptor** [zen-browser/desktop#2079] [Ideas]
### Captchas

- [X] I have read the instructions.
- [X] I have searched existing issues and avoided creating duplicates.
- [ ] I am not filing an enhancement request.

### What happened?

I do know that there is f6 key which does the exact same thing. But that's not what I want. I've been accustomed to using esc to get out of the address bar with other chromium browsers. Well there are means of mapping the f6 key to esc key but that will break the shortcuts of many websites and extensions that rely on esc key.
It would be great to have this integrated into zen browser as it will reduce the friction between those who come from chromium based browsers. F6 key is not that intuitive to something like getting out of address bar anyways.

### Reproducible?

- [ ] I have checked that this issue cannot be reproduced on Mozilla Firefox.

### Version

1.0.1-a.8 (64-bit) (Generic/Non-Portable)

### What platform are you seeing the problem on?

Windows

**Comment by linear** [comment]
<p><a href="https://linear.app/zen-browser/issue/ZEN-1977/esc-key-does-not-get-me-out-of-address-bar">ZEN-1977 'Esc key' does not get me out of address bar</a></p>

**Comment by atharva-p** [comment]
i think this a more of a feature request (or an enhancement) rather than an issue. if it is, considering closing this issue, and maybe give feedback on zen's discord

**Comment by nanoraptor** [comment]
It is a duplicate of #1142. Hence I'm closing this.
